packages/suite/src/actions/settings/deviceSettingsActions.ts

```diff
@@ -108,9 +108,7 @@ export const resetDevice = (params: ResetDevice = {}) => async (
         ...defaults,
         ...params,
     });
-    if (result.success) {
-        dispatch(addToast({ type: 'settings-applied' }));
```

Comment:
only used in onbaording, we probably don't want to see 'settings-applied' toast there